pkg/activator/handler/handler.go

```diff
@@ -61,7 +69,53 @@ func (a *ActivationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	err := a.Throttler.Try(revID, func() {
-		attempts, httpStatus := a.proxyRequest(w, r, target)
+		var (
+			attempts   int
+			httpStatus int
+		)
+		// If a GET probe interval has been configured, then probe
+		// the queue-proxy with our network probe header until it
+		// returns a 200 status code.
+		success := (a.GetProbeCount == 0)
+		if !success {
+			probeReq := &http.Request{
+				Method:     http.MethodGet,
+				URL:        target,
+				Proto:      r.Proto,
+				ProtoMajor: r.ProtoMajor,
+				ProtoMinor: r.ProtoMinor,
+				Host:       r.Host,
+				Header: map[string][]string{
+					http.CanonicalHeaderKey(network.ProbeHeaderName): []string{"true"},
+				},
+			}
+			settings := wait.Backoff{
+				Duration: 100 * time.Millisecond,
+				Factor:   1.3,
```

Comment:
Exponential backoff here seems backwards to what you'd actually want. If we're waiting for something to become ready, it's more likely to succeed as time goes on, so wouldn't we want to probe with exponentially decaying backoff (Factor < 1)?

Reply:
At a minimum, it's consistent with what we do today.  We can revisit this in a subsequent change.